maat-court-data-api/src/main/java/gov/uk/courtdata/repository/CrownCourtProcessingRepository.java

```diff
@@ -30,4 +30,8 @@ void invokeUpdateSentenceOrderDate(
             @Param("p_sentence_order_date") LocalDate sentenceOrderDate);
 
     List<RepOrderCCOutComeEntity> findByRepOrder_Id(Integer id);
+
+    void deleteAllByRepOrder_IdIn(List<Integer> repIdList);
```

Comment:
it's not typical to include underscores in method names.

Reply:
I'm not sure if it's actually to do with the repository naming structure in below class...

Reply:
The _ is to allow us to access the id inside the repOrder entity on RepOrderCCOutComeEntity.

Reply:
ah ok thanks for explaining!